singularity-eos/eos/modifiers/eos_unitsystem.hpp

```diff
@@ -175,13 +175,17 @@ class UnitSystem {
     Real srho = rho_unit_ * rho;
     switch (t_.PreferredInput()) {
     case thermalqs::density | thermalqs::temperature:
+      {
```

Comment:
The compiler (`gcc@9.3.0`) didn't like the `break` without an explicit score - likely the compiler can't guarantee that inlining `FillEos` of the templated type of `t_` is properly scoped, leading to an ambiguous `break` (e.g. `for-break` rather than `case-break`)